sequencer/src/bin/state-relay-server.rs

```diff
@@ -0,0 +1,43 @@
+use async_compatibility_layer::logging::{setup_backtrace, setup_logging};
+use clap::Parser;
+use sequencer::state_signature::relay_server::run_relay_server;
+
+#[derive(Parser)]
+struct Args {
+    /// Port to run the server on.
+    #[clap(
+        short,
+        long,
+        env = "ESPRESSO_STATE_RELAY_SERVER_PORT",
+        default_value = "8083"
+    )]
+    port: u16,
+
+    /// Threshold to form an available state signature package.
+    /// WARNING: this is a temporary flag, should remove after integrating with stake table.
+    /// Related issue: [https://github.com/EspressoSystems/espresso-sequencer/issues/1022]
+    #[clap(
+        short,
+        long,
+        env = "ESPRESSO_STATE_SIGNATURE_WEIGHT_THRESHOLD",
+        default_value = "3"
+    )]
+    threshold: u64,
+}
+
+#[async_std::main]
+async fn main() {
+    setup_logging();
+    setup_backtrace();
+
+    let args = Args::parse();
+
+    tracing::info!("starting state relay server on port {}", args.port);
+    run_relay_server(
+        None,
+        args.threshold,
+        format!("http://0.0.0.0:{}", args.port).parse().unwrap(),
```

Comment:
Remove `http://`, the server uses HTTP no matter what, and some platforms don't like this when they parse the socket address

Reply:
removed in [1dde57c](https://github.com/EspressoSystems/espresso-sequencer/pull/1016/commits/1dde57c61e4683b1fde67bb5bde9e666a563311b)

Reply:
Wait, server wont start after removing `http://`
```bash
state-relay-server-1   | Message:  called `Result::unwrap()` on an `Err` value: RelativeUrlWithoutBase
state-relay-server-1   | Location: sequencer/src/bin/state-relay-server.rs:39
state-relay-server-1   | 
state-relay-server-1   | Backtrace omitted. Run with RUST_BACKTRACE=1 environment variable to display it.
state-relay-server-1   | Run with RUST_BACKTRACE=full to include source snippets.

```

Reply:
I think we should keep this `http://`

Reply:
Oh maybe since you're parsing it as a URL, it's ok. The place where this was causing problems was just using a string, which was getting interpreted (wrongly) somewhere inside the web server library